### PR TITLE
New: Turkish translate

### PR DESCRIPTION
**Pull request by FabrieI**
https://github.com/gujjwal00/avnc/commit/5861f3d618c705f01d028c23267623fd4e2182a7

https://github.com/gujjwal00/avnc/commit/f1e409a94dbb07f6a6edc32f5869fe62a5a0df1a

Thanks again

### CODE REVIEW

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    <string name="msg_server_profile_deleted">Sunucu silindi</string>
+    <string name="msg_pip_not_supported">Bu cihazda desteklenmiyor</string>
+    <string name="msg_ptr_hiding_not_supported">Bu cihazda desteklenmiyor</string>
+    <string name="msg_loading">Yükleniyor…</string>
+    <string name="msg_imported">İçeri aktarıldı</string>
+    <string name="msg_exported">Dışarı aktarıldı</string>
+    <string name="msg_export_auth_required">Sunucuları dışa aktarmak için kimlik doğrulaması yapın</string>
+    <string name="msg_export_auth_missing">Cihazınız güvenli değil. Dışa aktarmadan önce kimlik doğrulamasını etkinleştirmek için ekran kilidini açın.</string>
+    <string name="msg_invalid_key_file">Geçersiz özel anahtar dosyası</string>
+    <string name="msg_invalid_vnc_uri">Geçersiz VNC URI'si</string>
```
Comment (gujjwal00) [inline app/src/main/res/values-tr/strings.xml:59]:
```suggestion
    <string name="msg_invalid_vnc_uri">Geçersiz VNC URI\'si</string>
```

**Review by FabrieI — COMMENTED** [review]
error fixed > Update strings.xml

**Comment by gujjwal00** [comment]
Thank you @FabrieI

**Comment by gujjwal00** [comment]
Hi @FabrieI, It looks like someone else has also added Turkish translation on Weblate now: https://hosted.weblate.org/projects/avnc/app/tr/

**Comment by FabrieI** [comment]
Hi, @gujjwal00 

I just saw it,ersen translation is more fluent.

**Pull request merged**